README.md

```diff
@@ -329,6 +329,18 @@ is still in development.  More details to follow.  The basic design is that one
 desired k3s and k3OS versions, plus their configuration and the operator will roll that out to
 the cluster.
 
+### Manual Upgrades
+
+For single-node or development use cases where the operator is not being used, you can upgrade the rootfs and kernel with the following commands. If you do not specify K3OS_VERSION, it will default to the latest release.
+
+```
+export K3OS_VERSION=v0.4.0
+/sbin/k3os-upgrade-rootfs
```

Comment:
I think it would be clarifying here to call out that the kernel upgrade is only available on non-overlay installations (aka it doesn't work on RPI).

Reply:
Done.